### PR TITLE
Upgrade to `polkadot-v1.1.0`

### PR DESCRIPTION
**Pull request by dastanfv**
Reasons to do the upgrade:

- dependencies are now using `polkadot-sdk` repo. 
- `frontier` released new tools, utils for writing and testing precompiles https://github.com/paritytech/frontier/tree/polkadot-v1.1.0/precompiles/src/testing
- there were some critical bug fixes on the `frontier` client side (`pending_runtime_api_inherents`)

Some notable changes:

- `polkadot` has updated `xcm` pallets with adding `staging-` prefix to it
- I renamed our precompile util crates with adding `laos-` prefix to it, since it would conflict with the `frontier` crates that we will be using

### CODE REVIEW

**Comment by dastanfv** [comment]
this should be fixed later
https://github.com/freeverseio/laos/issues/182

File: ownership-chain/node/src/command.rs [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
+	eth::db_config_dir,
+	service::new_partial,
 };
 
 fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 	Ok(match id {
-		"arrakis" => Box::new(chain_spec::ChainSpec::from_json_bytes(
-			&include_bytes!("../../specs/arrakis-frontier.json")[..],
-		)?),
-		"caladan" => Box::new(chain_spec::ChainSpec::from_json_bytes(
```
Comment (asiniscalchi) [inline ownership-chain/node/src/command.rs:28]: why Caladan network is removed ?

File: ownership-chain/node/src/command.rs [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
-			&include_bytes!("../../specs/caladan-raw.json")[..],
-		)?),
 		"dev" => Box::new(chain_spec::development_config()),
 		"template-rococo" => Box::new(chain_spec::local_testnet_config()),
 		"" | "local" | "local-v" => Box::new(chain_spec::local_testnet_config()),
```
Comment (asiniscalchi) [inline ownership-chain/node/src/command.rs:34]: what is local-v ?

**Review by asiniscalchi — CHANGES_REQUESTED** [review]

File: ownership-chain/node/src/command.rs (hunk 2 above)
Comment (dastanfv) [reply ownership-chain/node/src/command.rs:34]: it was needed to recognise our chain as a chain with `unified account` in the zombienet config. it was a temporary fix that we did when we were migrating to unified accounts. Without this, we won't be able to run `zombienet` locally

File: ownership-chain/node/src/command.rs (hunk 1 above)
Comment (dastanfv) [reply ownership-chain/node/src/command.rs:28]: good catch

File: ownership-chain/node/src/command.rs (hunk 2 above)
Comment (dastanfv) [reply ownership-chain/node/src/command.rs]: https://github.com/freeverseio/laos/pull/28

**Review by asiniscalchi — DISMISSED** [review]

**Review by asiniscalchi — APPROVED** [review]

**Pull request merged**